sdk/nodejs/package.json

```diff
@@ -21,8 +21,6 @@
         "require-from-string": "^2.0.1",
         "semver": "^6.1.0",
         "source-map-support": "^0.5.6",
-        "ts-node": "^7.0.1",
-        "typescript": "~3.7.3",
```

Comment:
discussion in https://github.com/pulumi/pulumi/pull/10122 around breaking changes.